Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ghcr.io/digininja/dvwa:latest
+
+RUN apt update \
+ && apt install -y --no-install-recommends \
+                mariadb-server socat net-tools iproute2 \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN service mariadb start \
+ && mariadb -uroot -e "CREATE DATABASE dvwa;" \
+ && mariadb -uroot -e "GRANT ALL ON *.* TO 'dvwa'@'%' IDENTIFIED BY 'p@ssw0rd' WITH GRANT OPTION;"
+
+EXPOSE 80 3306
+
+COPY start.sh /start.sh
+
+CMD ["/start.sh"]
```

Comment:
Just one minor suggestion: can we rename this startup script to `docker-entrypoint.sh` ? I'm seeing this "undocumented standard" on some of the official images and I'm trying to use the same approach on our projects. Just to make it easy to troubleshoot in the future.

Reply:
Great point! Just did it :)